test-runner/chrome/content/loggest-chrome-runner.js

```diff
@@ -1306,18 +1237,41 @@ function DOMLoaded() {
 
     try {
       runTests(configData).then(function(summaries) {
-        dump('\n\n***** ' + summaries.length + ' tests run: *****\n\n');
+        dump('\n************************\n\n');
+
         summaries.forEach(function(summary) {
           printTestSummary(summary);
         });
-        dump('\n************************\n\n');
+
+        dump('\n************************\n\n\n');
+        dump(' ╔══════════════════════════════════════════════════════════════╗\n');
+        dump(' ║ Run "make results" to view detailed results in your browser. ║\n');
+        dump(' ╚══════════════════════════════════════════════════════════════╝\n');
+        dump('                                                                 \n');
+        dump('           (Run "make clean" to delete old test logs.)           \n\n\n');
 
         var testResult = getTestResult(summaries);
         var jsonString = JSON.stringify({ result: testResult });
 
-        writeFile('test-logs', 'test-run.summary', jsonString).then(function() {
-          quitApp();
-        });
+        var INDEX_FILENAME = 'index.json';
+
+        readFile('test-logs', INDEX_FILENAME)
+          // Blah, file origins. Remove the JSONP function call before JSON-parsing.
```

Comment:
And this explains the above.  But this comment is not enough guilt!  It needs to be everywhere! ;)

So, Chrome doesn't like it, but this can be made to work in Firefox.  Here's a commit that does it:
https://github.com/asutherland/gaia-email-libs-and-more/commit/fc05a4882d5b3275b09ef479f09b8d62b335cc65

https://developer.mozilla.org/en-US/docs/Same-origin_policy_for_file:_URIs lets us do it as long as test-logs is a sub-directory of where index.html is.  So I inverted the symlink relationship.

The chromium issue is https://code.google.com/p/chromium/issues/detail?id=47416.  Despite Chrome not liking the file URI thing, I think one can make the argument that since GELAM currently has some Firefox-specific dependencies right now, it's not insane to say that you need to serve on localhost or just use firefox.  We can certainly have an easy node webserver for the chrome/whatever case.